src/actions/breakpoints.js

```diff
@@ -191,6 +191,34 @@ export function toggleAllBreakpoints(shouldDisableBreakpoints: boolean) {
   };
 }
 
+export function remapBreakpoints(sourceId: string) {
+  return async ({ dispatch, getState, sourceMaps }: ThunkArgs) => {
+    const breakpoints = getBreakpoints(getState());
+    const newBreakpoints = await remapLocations(
+      breakpoints,
+      sourceId,
+      sourceMaps
+    );
+
+    return dispatch({
+      type: "REMAP_BREAKPOINTS",
+      breakpoints: newBreakpoints
+    });
+  };
+}
+
+function remapLocations(breakpoints, sourceId, sourceMaps) {
```

Comment:
does `async function` work?

If so, can we add an `await` here. I'm guessing no because I bet the `Promise.all` was a hack lol